iceberg/iceberg-handler/src/test/queries/positive/iceberg_major_compaction_partition_evolution.q

```diff
@@ -31,7 +31,7 @@ create table ice_orc (
  )
 partitioned by (company_id bigint)
 stored by iceberg stored as orc 
-tblproperties ('format-version'='2', 'hive.compactor.worker.pool'='iceberg');
+tblproperties ('format-version'='2', 'hive.compactor.worker.pool'='iceberg','compactor.threshold.target.size'='1500');
```

Comment:
do we support units like 10mb or something?

Reply:
This value is in bytes, I did it like in Amoro config which does not have units. As discussed, we will maybe improve it later and support for units.